nifi-nar-bundles/nifi-extension-utils/nifi-prometheus-utils/src/main/java/org/apache/nifi/prometheus/util/BulletinMetricsHolder.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.prometheus.util;
+
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Gauge;
+
+public class BulletinMetricsHolder {
+
+    private CollectorRegistry registry = new CollectorRegistry();
+
+    public BulletinMetricsHolder() {
+    }
+
+    public BulletinMetricsHolder(final CollectorRegistry registry) {
+        this();
+        this.registry = registry;
+    }
+
+    public CollectorRegistry getRegistry() {
+        return registry;
+    }
+
+    public final Gauge BULLETIN = Gauge.build()
```

Comment:
Would also recommend instead of accessing gauges or counters directly that callers use methods which then can set labels and inc/dec/set values as needed.  This may also allow for isolated testing of the holder/registry.